output/cloud/expv2/output.go

```diff
@@ -275,14 +287,36 @@ func (o *Output) flushMetrics() {
 	o.logger.WithField("t", time.Since(start)).Debug("Successfully flushed buffered samples to the cloud")
 }
 
+func (o *Output) runFlushRequestMetadatas() {
+	t := time.NewTicker(o.config.TracesPushInterval.TimeDuration())
+
+	for i := int64(0); i < o.config.TracesPushConcurrency.Int64; i++ {
+		o.wg.Add(1)
+		go func() {
+			defer func() {
+				t.Stop()
+				o.wg.Done()
+			}()
+
+			for {
+				select {
+				case <-t.C:
+					o.flushRequestMetadatas()
+				case <-o.stop:
+					return
+				case <-o.abort:
```

Comment:
```suggestion
				case <-o.stop, <-o.abort:
```

Reply:
Are you sure this is a valid syntax?

Reply:
Hmm ... I guess it isn't as it needs to be statements instead of expressions as in the case of `switch` ... I think I have fallen for this more then once at this point 

Sorry :bow: